content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -404,6 +406,71 @@ instances:
 ### Serverless environment
 
 For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][15].
+
+### OpenTelemetry
+
+When using OpenTelemetry, map the following [resource attributes][16] to their corresponding Datadog conventions:
+
+| OpenTelemetry convention | Datadog convention |
+| --- | --- |
+| `deployment.environment` | `env` |
+| `service.name` | `service` |
+| `service.version` | `version` |
```

Comment:
So this is documented here https://docs.datadoghq.com/opentelemetry/schema_semantics/semantic_mapping/?tab=datadogexporter, I am fine with having it in both places but just wanted to flag it in case we prioritize avoiding duplication

Reply:
Right, this is something I was debating myself. Generally we want to avoid duplication, but in this case, I think the minor overhead of having this small table in two places outweighs the inconvenience we'd create pointing users to the existing table on another page. We could use a reusable snippet, but I think that might overengineering for something so small. Let's leave this for now, and we can adjust if it becomes a problem. 👍

Reply:
Okay, makes sense!